.github/workflows/maven.yml

```
@@ -21,29 +21,44 @@ jobs:
+            test-jvm: -Djvm=$JAVA_HOME_11_X64/bin/java
+          - run-java: '17'
+            build-java: '17'
+            module: tests-v24
+            test-jvm: ''
+          - run-java: '21'
+            build-java: '21'
+            module: tests-v25
+            test-jvm: ''
+    name: JDK ${{ matrix.run-java }}
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)